### DISCUSSION TITLE
h-max and h-min are missing

### DISCUSSION THREAD

**Discussion by quansenB** [tailwindlabs/tailwindcss#5735] [Ideas]
I'm wondering why h-min (min-content) and h-max (max-content) are missing, as I am using this all the time right now through the jit-[]-notation. 

Is there any reason for it?

If not it would be nice to have it by default

**Comment by lukewarlow** [comment]
I had the same thought so added them in a [PR](https://github.com/tailwindlabs/tailwindcss/pull/5729). Which has been merged so these utilities will be in 3.0.

**Reply by simonswiss** [reply]
Thanks for this @lukewarlow 🙌
